coffea/nanoevents/factory.py

```diff
@@ -33,7 +40,7 @@ def __setstate__(self, state):
         self._events = lambda: None
 
     @classmethod
-    def from_file(
+    def from_root(
```

Comment:
We should have a stub `from_file` that calls `from_root` and deprecate it for 0.8